source/introduction/advantages.rst

```diff
@@ -23,19 +23,19 @@ Advantages for system administrators
+Application authors may want to reach as many users as possible, regardless of the operating system and version users are running.  With the traditional model, application authors either need to get their application into Linux distributions by creating packages according to the rules of distributions, which can be a lengthy and time-consuming process, or need to find someone who creates the application package for them. It does not help that different Linux distributions have very different rules for packaging applications. This means that if an author is successful to package an application according to the rules of one distribution, it may well not be suitable for inclusion in another distribution. In any case, it is not a quick and easy operation to get a new application into every Linux distribution. And even if an author gets an application into a Linux distribution, it will only appear in the soon-to-be-released version of that Linux distribution, which means that all the currently existing users using older versions of the Linux distribution are left out in the cold.
 
 If a third party (the distribution's "maintainer") packages the application for inclusion in the distribution, the third party may make unintended changes to the application not authorized by the original author. This has led to some application authors to ask Linux distributions to no longer distribute their software because they did not like the changes.
 
-Some application offers do not want their applications to be distributed randomly, but be in full control over who downloads what and when. For example and application author may want users to fill in a survey before downloading the application. Also, the author may want to count the number of downloads.  In other cases still, the application author may want to ask for a donation or even her required payment before the user can download the application through services such as Gumroad, PayPal, or Patreon. In the traditional Linux distribution model, this is not possible.
+Some developers do not want their applications to be distributed randomly, but be in full control over who downloads what and when. For example an application author may want users to fill in a survey before downloading the application. Also, the author may want to count the number of downloads.  In other cases still, the application author may want to ask for a donation or even a required payment before the user can download the application through services such as Gumroad, PayPal, or Patreon. In the traditional Linux distribution model, this is not possible.
```

Comment:
I think that was meant to be "application authors", but "developers" is just fine IMO.

Reply:
Yes, I also thought that, but changed it to developers so it is more straight forward. Can be changed though for consistency.